src/Nox.ClientApi.Tests/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Infrastructure.Persistence/AppDbContext.g.cs

```diff
@@ -71,6 +71,7 @@ NoxCodeGenConventions codeGeneratorState
     public DbSet<ClientApi.Domain.StoreLicense> StoreLicenses { get; set; } = null!;
 
 
+public DbSet<ClientApi.Domain.WorkplaceLocalized> WorkplacesLocalized { get; set; } = null!;
```

Comment:
Why is this committed, is it because of the Merge Conflicts?

Reply:
No. It's committed because the workplace has a localized description field, which means there will be a WorkplaceLocalized entity in the AppDbContext and WorkplaceLocalizedDto in DtoDbContext.